DemoViper/VIPER/Router.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Router.swift
+//  viper
+//
+//  Created by Su Van Ho on 9/8/19.
+//  Copyright © 2019 Nimble Co. Ltd. All rights reserved.
+//
+
+import UIKit
+
+/// These protocols is defined for people who want to take a look on a big picture of VIPER, optional for using.
+///
+/// Router contains navigation logic for describing which screens are shown in which order
+protocol Router: class {
```

Comment:
`class protocol` is deprecated, let's change it to `AnyObject` protocol

https://stackoverflow.com/questions/30176814/whats-the-difference-between-a-protocol-extended-from-anyobject-and-a-class-onl

Reply:
Fixed